src/functions/template-tags/general.php

```diff
@@ -1518,48 +1527,60 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null, $skip
 		 */
 		$remove_shortcodes = apply_filters( 'tribe_events_excerpt_shortcode_removal', true );
 
-		// Get the Excerpt or content based on what is available
-		$excerpt = has_excerpt( $post->ID ) ? $post->post_excerpt : $post->post_content;
+		$cache_excerpts_key = implode( ':', [
+			$post->ID,
+			$skip_postdata_manipulation,
+			$allow_shortcodes,
+			$remove_shortcodes,
+			json_encode( $allowed_html )
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 91e494b8bd3bc176e4511a8ded11f5bf</sub>